test/testcafe/framework/page-objects/YubiKeyAuthenticatorPageObject.js

```diff
@@ -0,0 +1,23 @@
+import BasePageObject from './BasePageObject';
+
+export default class YubiKeyAuthenticatorPageObject extends BasePageObject {
+  constructor(t) {
+    super(t);
+  }
+
+  verifyFactor(name, value) {
+    return this.form.setTextBoxValue(name, value);
+  }
+
+  getPageTitle() {
+    return this.form.getElement('.okta-form-title').textContent;
+  }
+
+  getPageSubtitle() {
+    return this.form.getElement('.okta-form-subtitle').textContent;
+  }
+
+  submit() {
+    return this.form.clickSaveButton();
```

Comment:
I think they are in base page object itself

Reply:
Not that I could find. It looks like the `BasePageObject` provides helpers for header/footer interactions, as opposed to the body content.

I [created a ticket](https://oktainc.atlassian.net/browse/OKTA-410074) to have a `BaseAuthenticatorPageObject`, which should help us reduce the number of extended classes. I'll plan on taking this on next week.

Reply:
I can find methods for title and subtitle. 

https://github.com/okta/okta-signin-widget/blob/master/test/testcafe/framework/page-objects/BasePageObject.js#L69

Reply:
Ah - somehow I overlooked those. Updated!